### ISSUE TITLE
Post 1.0 development plans

### ISSUE THREAD

**Issue by eliotjordan** [geoblacklight/geoblacklight#439]
Goals, features, and ideas for GeoBlacklight development after a 1.0 release. The following items were copied over from #351:
- [x] Index maps
- [ ] Grouping around 'collections'
- [ ] ~~Multiple direct file downloads per record~~ waiting on additional information
- [ ] Improved accessibility
- [x] More documentation & blog posts
- [x] ~~Leaflet 1.0~~ wait until it is released

**Comment by sgbalogh** [comment]
How about https://github.com/geoblacklight/geoblacklight/issues/412 and https://github.com/geoblacklight/geoblacklight/issues/449 ?

- [ ] Support for additional constants
- [ ] Simple data relation tracking

**Comment by eliotjordan** [comment]
We're at 2.0. No longer needed.
